File: Algorithms/Python/0001 Two Sum/Two_Sum.py
# Problem :    https://leetcode.com/problems/two-sum/
# Solution:    https://github.com/tjzhym/LeetCode-solution
# Author :     zhym (tjzhym)
# Date   :     2021-5-28




class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        nums_len = len(nums)
        for i in range(nums_len):
            for j in range(i+1,nums_len):
                if (nums[i] + nums[j] == target):
                    return [i,j]
                    break

File: Algorithms/Java/0001 Two Sum/Two Sum.java
// Problem :    https://leetcode.com/problems/two-sum/
// Solution:    https://github.com/tjzhym/LeetCode-solution
// Author :     zhym (tjzhym)
// Date   :     2021-7-11


class Solution {
    public int[] twoSum(int[] nums, int target) 
    {
        
        int nums_len = nums.length;
        for (int i = 0; i < nums_len-1; i++)
        {
            for (int j = i+1; j< nums_len; j++)
            {   
                if (nums[i] + nums[j] == target)
                {
                    return new int[]{i, j};
                }
                 
            } 
        }
        return new int[2];
    }
}
